kernel/realsense/d4xx.c

```diff
@@ -4967,26 +4967,29 @@ static int ds5_dfu_device_open(struct inode *inode, struct file *file)
 			state->client->adapter);
 #endif
 #endif
+	mutex_lock(&state->lock);
 	if (state->dfu_dev.device_open_count)
 		return -EBUSY;
```

Comment:
We need `mutex_unlock(&state->lock);` here right?

Reply:
right, this was another adjustment of where mutex should be hold and we missed that.